### PR TITLE
[JENKINS-35868] changeset url

### PR DESCRIPTION
**Pull request by imeredith**
**Decription**

Adds urls for changeset commits. ie to github.

**Submitter checklist**
- [x] Change is code complete and matches issue description

- [x] Apppropriate unit or acceptance tests or explaination to why this change has no tests

No extra tests for this as I can't make it generate a url when doing it locally.

**Reviewer checklist**
- [x] Run the changes and verified the change matches the issue description
- [x] Reviewed the code
- [x] Verified that the appropriate tests have been written or valid explaination given

@reviewbybees

### CODE REVIEW

File: blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ChangeSetResource.java [hunk 1]
```diff
@@ -50,6 +53,20 @@ public String getTimestamp(){
         }
     }
 
+    @Exported
+    public String getUrl() {
```
Comment (i386) [inline blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ChangeSetResource.java]: Does this mean its only ever available in the embedded driver?

Comment (imeredith) [inline blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ChangeSetResource.java]: Yes. All of that data is only in the embedded driver.

File: blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ChangeSetResource.java [hunk 2]
```diff
@@ -50,9 +50,38 @@ public String getTimestamp(){
         }
     }
 
+    @Override
+    public String getUrl() {
```
Comment (i386) [inline blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ChangeSetResource.java]: @imeredith is there a unit test worth adding here?

**Comment by michaelneale** [comment]
yeah I think a unit test would be nice here, but not sure if there is precedent for PRs in the multibranch test suite

File: blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ChangeSetResource.java (hunk 2 above)
Comment (imeredith) [inline blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ChangeSetResource.java]: I don't know how to unit test this. It's using a local repo only.

Comment (michaelneale) [inline blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ChangeSetResource.java]: fair enough

**Comment by michaelneale** [comment]
haven't had time to test this out but code looks good

**Comment by michaelneale** [comment]
🐝 tested out and works. Accept explanation. Nice work.

**Pull request merged**